Cargo.toml

```diff
@@ -17,15 +17,20 @@ opt-level = "z"
 default = ["std", "embassy", "esp-idf-svc/native"]
 
 pio = ["esp-idf-svc/pio"]
-std = ["alloc", "esp-idf-svc/binstart", "esp-idf-svc/std"]
-alloc = ["esp-idf-svc/alloc"]
-nightly = ["esp-idf-svc/nightly"]
-experimental = ["esp-idf-svc/experimental"]
+all = ["std", "nightly", "experimental", "embassy"]
```

Comment:
These are dependencies?

Reply:
Those are [Features](https://doc.rust-lang.org/cargo/reference/features.html). I'm not using any conditional compilation yet, but it's still nice way to logically categorize some of the dependencies. Lets you enable or disable sets of dependencies.